### ISSUE TITLE
RINO2 - FATAL ERROR: Uncaught crypto exception. Suppresssing output.

### ISSUE THREAD

**Issue by gitkald** [friendica/friendica#1917]
After upgrading my friendica to 3.4.2 - 1188 and activating RINO1, I get the fatal error from cron job (poller.php).

If I switch back to RINO1 all works fine.

**Comment by annando** [comment]
The fatal error is "FATAL ERROR: Uncaught crypto exception. Suppresssing output."

It is generated here: https://github.com/friendica/friendica/blob/master/library/defuse/php-encryption-1.2.1/Crypto.php#L662

The system should handle this error in a better way.

**Comment by fabrixxm** [comment]
we  shouldn't change the exception handler for motivations here https://github.com/friendica/friendica/blob/master/library/defuse/php-encryption-1.2.1/Crypto.php#L635-L643
I think we miss some test to check if the system can run the crypto library.

There is a test in the library. If it's possible, should be run on server, something like
```bash
$ cd /your/friendica/library/defuse/php-encryption-1.2.1/
$ bash test.sh
```
and report output

**Comment by tobiasd** [comment]
@gitkald could you check the results of that test script?

**Comment by tobiasd** [comment]
@gitkald have you had time to check the result of the test script mentioned by @fabrixxm?

**Comment by MrPetovan** [comment]
Closed by #4276
